src/ui-test/runtimes/spring/README.md

```diff
@@ -0,0 +1,26 @@
+# Spring Camel Rest example using exec:java
+
+This project is used as template when testing wsdl2rest vscode
+extension. To run this project you must do following steps.
+
+## Steps using wsdl2rest vscode extension
+
+1. Create or find wsdl file. It can be file in filesystem or url to file.
+2. Run the vscode extesions command.
```

Comment:
typo: "extesions" -> "extensions"?

Reply:
Simplified it to `2. Run extension command.`